src/js/common/config.js

```diff
@@ -26,7 +26,7 @@ export const defaultConfig = {
         local: false,
     },
     refreshTimeout: 5 * 60 * 60,
-    enableRemoteRules: !navigator.userAgent.match(/firefox/i),
+    enableRemoteRules: !navigator.userAgent.match(/firefox|safari/i),
```

Comment:
It seems that you've not checked what a chromium UA looks like...
Matching `safari` effectively disables remotes rules for Chromium too.

https://developers.whatismybrowser.com/useragents/explore/software_name/chromium/